packages/core/test/metrics/MetricsServiceTest.ts

```diff
@@ -2,7 +2,8 @@ import { assert, expect } from "chai";
 import mocha from "mocha";
```

Comment:
This is under `test` folder already, then the file should be named the same as the file it is testing.

Reply:
Then it's hard to get if you working with test or actual file. Mockito do not provide valid stacktraces so it's also hard to get where error happened.